kodilla-patterns2/src/main/java/com/kodilla/kodillapatterns2/adapter/bookclasifier/libraryb/BookB.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.kodillapatterns2.adapter.bookclasifier.libraryb;
+
+public class BookB {
```

Comment:
Jakie BookB? Jak masz dwie biblioteki javowe w postaci jara, to nie mozesz zmienic nazw klas.